pkg/distributors/test/MultiDistributor.test.ts

```diff
@@ -885,359 +903,391 @@ describe('MultiDistributor', () => {
       await distributor.fundDistribution(anotherDistribution, DISTRIBUTION_SIZE, { from: distributionOwner });
     });
 
-    context('when the user did specify some amount', () => {
-      const amount = fp(1);
-
-      context('when the user has previously staked the requested balance', () => {
-        sharedBeforeEach('stake amount', async () => {
-          await stakingTokens.mint({ to: user1, amount });
-          await stakingTokens.approve({ to: distributor, amount, from: user1 });
+    const itHandlesUnstaking = (unstake: (token: Token, amount: BigNumberish) => Promise<ContractTransaction>) => {
+      context('when the user did specify some amount', () => {
+        const amount = fp(1);
 
-          await distributor.stake(stakingToken, amount, { from: user1 });
-        });
+        context('when the user has previously staked the requested balance', () => {
+          sharedBeforeEach('stake amount', async () => {
+            await stakingTokens.mint({ to: from, amount });
+            await stakingTokens.approve({ to: distributor, amount, from: from });
 
-        const itTransfersTheStakingTokensToTheUser = () => {
-          it('transfers the staking tokens to the user', async () => {
-            await expectBalanceChange(() => distributor.unstake(stakingToken, amount, { from: user1 }), stakingTokens, [
-              { account: user1, changes: { [stakingToken.symbol]: amount } },
-              { account: distributor.address, changes: { [stakingToken.symbol]: amount.mul(-1) } },
-            ]);
+            await distributor.stake(stakingToken, amount, from, from, { from });
```

Comment:
from from (from). from? from: from

Reply:
I'm not quite sure what you'd prefer here as while it's very frommy it's correct. :laughing:

I can stake some tokens for `from` from `other` to potentially make it clearer that we're just getting "some staked tokens" and it doesn't really matter where from. Is that your concern or just checking based on the other `to/from` issues?

Reply:
Nah I think this is fine, the line just struck me as funny :joy: